renderdoc/driver/ihv/intel/driver_store_path.cpp

```diff
@@ -0,0 +1,383 @@
+/*****************************************************************************\
```

Comment:
These files need to be re-organised a bit. Any files (usually headers but sometimes source) that come directly from an upstream source and are preserved originally should go under an `official/` subdirectory with a `README.md` or similar pointing to where they can be found. Looking around these files _seem_ to be the same as the files [in this opencl repository](https://github.com/intel/opencl-intercept-layer/tree/master/mdapi) except split into header/source. I'm not sure if that's the official source of them though it seems kind of incidental - [the metrics_discovery repository](https://github.com/intel/metrics-discovery) seems more likely but it doesn't contain the driver store path files.

I'd prefer that they're left as-is rather than split, since then it's clear these are the identical files. Either way putting them under `official/` means that clang-formatting isn't applied to foreign code (and I won't review any of the contents) and it makes the license applying to these separate files more clear.

Code written for renderdoc specifically (i.e. in this case `intel_counters.cpp/h`) stays where it is.